Osiris/Hacks/Misc.cpp

```diff
@@ -383,6 +386,60 @@ void Misc::drawBombTimer() noexcept
 
     ImGui::textUnformattedCentered(ss.str().c_str());
 
+    //if (localPlayer) {
+    float finalBombDamage = 0.f;
+
+    for (auto i = interfaces->engine->getMaxClients(); i <= interfaces->entityList->getHighestEntityIndex(); i++)
```

Comment:
this is for idiots, ye?🤦🏻‍♂️
https://github.com/danielkrupinski/Osiris/blob/2135abad64fde6f45e14bdbee49cf0fbe24c029f/Osiris/GameData.cpp#L261-L264

Reply:
> 
> 
> this is for idiots, ye?🤦🏻‍♂️
> 
> https://github.com/danielkrupinski/Osiris/blob/2135abad64fde6f45e14bdbee49cf0fbe24c029f/Osiris/GameData.cpp#L261-L264

how to get the entity???

Reply:
just store bomb handle
```
void BombData::update() noexcept
{
    if (memory->plantedC4s->size > 0 && (!*memory->gameRules || (*memory->gameRules)->mapHasBombTarget())) {
        if (const auto bomb = (*memory->plantedC4s)[0]; bomb && bomb->c4Ticking()) {
            blowTime = bomb->c4BlowTime();
            timerLength = bomb->c4TimerLength();
            defuserHandle = bomb->c4Defuser();
            handle = bomb->handle();
            if (defuserHandle != -1) {
                defuseCountDown = bomb->c4DefuseCountDown();
                defuseLength = bomb->c4DefuseLength();
            }

            if (*memory->playerResource) {
                const auto& bombOrigin = bomb->origin();
                bombsite = bombOrigin.distTo((*memory->playerResource)->bombsiteCenterA()) > bombOrigin.distTo((*memory->playerResource)->bombsiteCenterB());
            }
            return;
        }
    }
    blowTime = 0.0f;
}
```